plugins/example-site/src/site/_site/index.html

```diff
@@ -0,0 +1,6 @@
+<html>
+  <head>
+    <title>Page title</title>
+  </head>
+  <body>Page body</body>
```

Comment:
I dont know how to do it. I'm a dummy. @clintongormley made these simple rest tests for the cloud plugins that just checked they were installed, and i used that logic here so we have something.